### DISCUSSION TITLE
Integration of WebGL content (e.g. SketchFab)

### DISCUSSION THREAD

**Discussion by Sangrail** [outline/outline#1543] [General] [answered]
Hi there,

am looking for help with integrating WebGL content with this platform - e.g. https://skfb.ly/6BPP8. Any pointers?

**Comment by tommoor** [comment, marked as answer]
Embeds are currently added on a per-application basis, so we'd need to create an embed specifically for Sketchfab, you can see all the existing ones here for an example on how to create:
https://github.com/outline/outline/tree/develop/app/embeds
